apps/extension/server/src/server.ts

```diff
@@ -259,6 +265,11 @@ async function handleDocumentChange(textDocument: TextDocument): Promise<void> {
       });
     }
   } catch (e) {
+    if (isErrorWithMessage(e)) {
+      connection.sendNotification('extensionError', {
+        message: e.message,
+      });
+    }
```

Comment:
If we don't trust the thrown thing here to be an error then we should also handle the other case. Something like this would do the trick:

```suggestion
    connection.sendNotification('extensionError', {
      message: isErrorWithMessage(e) ? e.message : 'Unknown error',
    });
```